src/test/java/com/github/princesslana/smalld/TestIdentify.java

```diff
@@ -13,6 +16,9 @@
 
   private MockDiscordServer server;
 
+  private static final BiConsumer<WebSocket, Response> SEND_HELLO =
```

Comment:
This seems like it shouldn't be a `BiConsumer`... making it a `BiConsumer` just so it looks pretty doesn't seem like a good reason to do it, it only uses the `websocket` so why are we also passing in `r`, we could just below do `server.gateway().onOpen((ws, r) -> SEND_HELLO.consume(ws))` It kind of makes sense for it to only be a `Consumer`. The argument "it takes up less memory / instantiates 1 instead of 2 objects" is kind of bland here as well as a micro-optimization for a test. Curious as to the reasoning. I'm not saying it's wrong or it shouldn't be that way, just curious.

Reply:
Just to add on, it's more readable as `Consumer` imo, since we `KNOW` for a fact that it only consumes one object.

Reply:
It's simply an extraction of repeated code, so that `server.gateway().onOpen(SEND_HELLO);` becomes readable. It's a BiConsumer of that type because that's what `onOpen` expects.

Reply:
alright, that's fair